content/news/2024/07/15/role-adverts.md

```diff
@@ -0,0 +1,87 @@
+
+## Communications lead
+
+### Role Overview
+As the Communications Lead for PyBaMM, you will be responsible for managing and growing the project's online presence, developing and executing our communication strategy, and creating engaging content that resonates with our audience. You will play a key role in defining PyBaMM's communication policy, supporting the development of educational materials, and promoting our annual conference. Your efforts will help to increase awareness, engagement, and support for PyBaMM within the scientific and engineering communities.
+
+### Duties and Responsibilities
+* Oversee and manage PyBaMM's current social media platforms, particularly X (Twitter) and LinkedIn.
+* Create engaging content regularly, starting with one post every other week and aiming to increase to one post per week in the mid-term.
+* Define and uphold the communication policy of the project, including evaluating and establishing a presence on additional platforms as needed.
+* Assist in the creation of educational materials, including video tutorials and other resources.
+* Collaborate with the conference organising committee to promote PyBaMM's annual conference, ensuring effective communication and outreach.
+
+### Criteria
+1. Proven experience in communication, preferably in science communication.
+2. Proficiency with X (Twitter) and LinkedIn is required. Experience with other social networks will also be valued positively.
+3. Experience with open-source software projects.
+4. Experience or understanding of battery modelling.
+5. Ability to work both independently and as part of a team.
+6. Skills in graphic design (desirable).
+7. Experience with video editing (desirable).
+8. Being a contributor or maintainer of PyBaMM (desirable).
+
+
+## Community manager
+
+### Role Overview
+As the Community Manager for PyBaMM, you will serve as the primary liaison between various teams and the broader community of users and contributors. Your role will include managing community interactions, reporting on community health, and overseeing internship programs. You will also be responsible for triaging GitHub discussion posts and monitoring the project's email accounts. Your efforts will help ensure a vibrant, collaborative, and supportive community around PyBaMM.
+
+### Duties and Responsibilities
+* Liaise and coordinate with the various teams within the PyBaMM community to facilitate smooth communication and collaboration.
+* Report to the Steering Council (SC) quarterly about the state of the community, covering aspects from user engagement to maintainer activities.
+* Act as the project’s contact point for internship programs, such as Google Summer of Code, ensuring successful participation and mentorship.
```

Comment:
We could mention Outreachy and GSoD here as well. It might also be useful to link to the open-source programs page (https://pybamm.org/open-source-programs/).

Reply:
I have added Outreachy (I thought if I added more it would end up a rather long list, and the point is to illustrate with a couple examples) and also added the hyperlink.